js-exercises/private-props/privateProps.js

```diff
@@ -0,0 +1,26 @@
+function privateProps(obj, isPrivate) {
+    return new Proxy(obj, {
+        set(target, prop, value) {
+            if(isPrivate(prop)) {
+                throw new TypeError("Can't set property "+prop);
+            }
+            else {
+                target[prop] = value;
+                return true;
+            }
+        },
+        has(target, prop) {
+            if(isPrivate(prop)) {
+                return false;
+            }
+            else true;
+        },
+        ownKeys(target) {
+            let allKeys = Object.keys(target);
```

Comment:
Since we are not initializing the variable `allKeys `again. Can we replace `let `with `const`.

Reply:
Yes, I will surely make that change!